java/client/build.gradle

```diff
@@ -180,7 +181,7 @@ publishing {
         mavenJava(MavenPublication) {
             from components.java
             groupId = 'software.amazon.glide'
-            artifactId = 'glide'
+            artifactId = 'glide-for-redis'
```

Comment:
glide-for-valkey?

Reply:
This name was given by Andrew

Reply:
@acarbonetto - is this right or should it get rebranded now?

Reply:
0.4.3 - is branded as glide for redis (so suggesting to keep it for consistency), 1.0.0 - will be branded differently.